proposals/proposal_modal.py

```diff
@@ -39,14 +34,14 @@ class ProposalModal(ui.Modal, title="Create/Edit Proposal"):
         max_length=2000,
     )
 
-    def __init__(self, channel, proposal):
+    def __init__(self, channel, proposal, proposal_type=None):
```

Comment:
does this need to have a "None" default value or can it just be a required field?

Reply:
> does this need to have a "None" default value or can it just be a required field?

Didn't need to be none and noticed an issue here when editing a proposal, I've updated this now to address that problem. Thanks for picking this up